mlc_llm/relax_model/llama.py

```diff
@@ -1516,7 +1553,8 @@ def get_model(args, hf_config):
             **hf_config,
             dtype=dtype,
             position_embedding_base=position_embedding_base,
-            combine_matmul=True,
+            # TODO: Re-enable with CombineParallelMatmul
+            combine_matmul=False,
```

Comment:
Is `CombineParallelMatmul` with lora supported now?

Reply:
Unfortunately, no.  It will require improvements to `LiftTransformParams`, in order to lift out a parameter transformation that can be used for every function in an `IRModule`.